Dockerfile

```diff
@@ -8,9 +8,8 @@
 # https://packagemanager.posit.co/cran/__linux__/jammy/2023-10-30
 
 # set proper base image
-ARG PLATFORM="linux/amd64"
 ARG R_VERS="4.3.1"
-FROM --platform=$PLATFORM rocker/r-ver:$R_VERS
```

Comment:
This is what allows me to build for arm64 which is significantly faster when running locally, so I'd prefer to keep this possibility.

Reply:
As I mentioned in a [comment](https://github.com/RMI-PACTA/workflow.pacta/pull/13#issuecomment-1816145917) on #13, the preferred mechanism for specifying alternate platforms is to declare those at build time, rather than in dockerfile.

Reply:
I think my fear here was... if one does *not* specify the platform explicitly in the build command, then Docker defaults to a system specific option, which then is not determined by what's in this repo and rather left up to chance. The goal with this was to enforce a proper default (linux/amd64) on all/any platform, but still allow for a user to explicitly choose/try something different.

Reply:
I guess this seemed more important if we would be building this locally on different machines, but if we're all in with building through GH Actions (totally in favor of that), then it becomes much less relevant 👍🏻

Reply:
@cjyetman with @20 open, can I resolve this thread?